tools/scylla-nodetool.cc

```diff
@@ -418,6 +449,29 @@ const std::map<std::string_view, std::string_view> option_substitutions{
 std::map<operation, operation_func> get_operations_with_func() {
 
     const static std::map<operation, operation_func> operations_with_func {
+        {
+            {
+                "cleanup",
+                "Triggers removal of data that the node no longer owns",
+R"(
+You should run nodetool cleanup whenever you scale-out (expand) your cluster, and
+new nodes are added to the same DC. The scale out process causes the token ring
+to get re-distributed. As a result, some of the nodes will have replicas for
+tokens that they are no longer responsible for (taking up disk space). This data
+continues to consume diskspace until you run nodetool cleanup. The cleanup
+operation deletes these replicas and frees up disk space.
+
+Fore more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/cleanup.html
+)",
+                {
+                    typed_option<int64_t>("jobs,j", "The amout of compaction jobs to be used for the cleanup (unused)"),
```

Comment:
typo: amount. But even better, "number".

Reply:
Anyway, I have to ask why do we want to support and silently ignore options - instead of not supporting them and failing if the user tries to use them.

Reply:
I think the idea is that any scriptology users might have around from their C* cluster, they can keep using when they switch to Scylla. So at least for options that don't make a material difference (like this --jobs option), we just silently swallow them. This is something @avikivity insists on. I personally don't enjoy porting these no-op commands and writing tests for them, just to check they do nothing.

Note that in case of unimplemented options that are supposed to trigger completely different behaviour, I did decide to go against this and reject them outright.

Reply:
These "noop" options make sense for things like performance-oriented **configuration options**, which is fine to ignore (we have better performance anyway). It is less fine, in my opinion, in options that were supposed to _do something_ originally, and now we just ignore them. Yes, the script will work, but will work incorrectly!

In the special case that there is a specific option which was already ignored in Cassandra, it is fine that we accept and ignore it as well. I don't know why anyone would have such an option in a script if it didn't work in Cassandra either, but what the heck...

Reply:
Well, I think this options (and all the similar ones) did and do work in C*, but they are ignored in our clone of the C* nodetool. Well, actually who knows, I didn't check the current C* nodetool.

Reply:
The typo or wrong word is still there, by the way.

Reply:
Typo fixed in v3.